action.yml

```diff
@@ -117,18 +105,25 @@ runs:
           -v \
           $(go list ./... | grep -v '${{ inputs.golang-unit-tests-exclusions }}')
     #
-    # Both unit and integration tests including code coverage.
+    # Unit & integration tests including code coverage.
     #
     - name: unit & integrations tests and code coverage
       shell: bash
       run: |
         go test \
-          --tags=integration \
-          -cover \
-          -covermode=atomic \
-          -coverprofile=coverage.txt \
+          -coverpkg=./... \
+          -coverprofile=profile.cov \
           -p ${{ inputs.golang-number-of-tests-in-parallel }} \
           -race \
           -short \
+          --tags=integration \
```

Comment:
This could be maybe be a parameter.

Reply:
Will require some investigation and documented the steps in: https://github.com/schubergphilis/mcvs-golang-action/issues/71